packages/app/cypress/e2e/runner/reporter.command_errors.cy.ts

```diff
@@ -285,7 +285,6 @@ describe('errors ui', {
     })
 
     verify('assertion failure in request callback', {
-      column: 22,
```

Comment:
Are you referring to test coverage or how code coverage calculation works?

Reply:
I took a bit of a further look into this and from what I can see we are trying to infer the row/column from the `eval`ed chai assert/expect which seems to be slightly off from what we expect. This leads to about a line or 2 difference when opening the file in the IDE vs what is displayed in the error console. I am not too sure how to fix this to be honest since it might be hard to get the accurate stack from the evaled context.

FWIW this is the case with typescript 5 usage and cypress today. Do we want to make an issue to look into correctly calculating the user invocation stack for typescript 5 users?

Reply:
Sure, lets create an issue

Reply:
created https://github.com/cypress-io/cypress/issues/29614 and linked FIXME comments in 307eb286fe